students/ikempcode33/Lesson 3/test_basic_operations.py

```diff
@@ -0,0 +1,85 @@
+"""Tests customer model operations"""
+
+import logging
+from customer_model import *
+from basic_operations import *
+from unittest import TestCase
+
+#Set up logging info
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+logger.info("logger is active_")
+
+
+class TestBasicOp(TestCase):
+    logger.info("testing basic operations")
+
+    def setUp(self):
+        #used to create database tables and set up the test customers data
+        logger.info('creating tables')
+        database.drop_tables([Customer])
+        database.create_tables([Customer])
+        logger.info('tables are created.')
+
+        logger.info('Adding data to database')
+        add_customer(1773, 'Karen', 'Smith',
+                     '142 42 st, Bellevue, WA 98105',
+                     '123-456-7890', 'ksmith99@gmail.com',
+                     40000.00, True)
```

Comment:
These are the last two parameters _add_customer()_ expects:

`status, credit_limit`

There's a problem with the order.